templates/borgmatic.service.j2

```diff
@@ -0,0 +1,60 @@
+# Managed by Ansible, please don't edit manually
+
+[Unit]
+Description=borgmatic backup
+Wants=backup_normal_repo.timer
+Wants=network-online.target
+After=network-online.target
+# Prevent borgmatic from running unless the machine is plugged into power. Remove this line if you
+# want to allow borgmatic to run anytime.
+ConditionACPower=true
+
+[Service]
+Type=oneshot
+User={{ borgbackup_user }}
+ExecStart=/usr/local/bin/borgmatic -c /etc/borgmatic/{{ borgmatic_config_name }}
```

Comment:
Can I use this as template service unit, so the same service could run multiple config files? I don't think your Systemd changes cover this case yet?

Reply:
Making this a separte issue for GSoC guys.

Reply:
Hi m3nu,

this was tricky, but i have a solution should i push it, or as a own PR?

Thanks Con

Reply:
Let's keep it for later. This PR is already too large. #115